src/dispatch/static/dispatch/src/incident/TimelineTab.vue

```diff
@@ -1,10 +1,25 @@
 <template>
   <v-container>
+    <v-switch v-model="show" label="Show details"></v-switch>
```

Comment:
nit: this would look a bit cleaner if it was right aligned, I think you could just add <v-spacer />